architect.yml

```diff
@@ -1,40 +1,62 @@
 name: nuxt
-description: A starter project showing how to make use of JSX with Typescript in Nuxt.js
+description: A starter project showing how to make use of Typescript in Nuxt.js
 keywords:
   - nuxt.js
   - typescript
-  - jsx
-  - tsx
+  - vue
 
+# dependencies:
+#   node-rest-api: latest
```

Comment:
Add this block right above the dependency block (just ensuring the line # matches with the changes):
```
## Want to learn about dependencies? Follow along with "Adding a Dependency" in the README
##   then uncomment the two lines below and uncomment line 41 (REACT_APP_API_ADDR)
```